tmate/install.sh

```diff
@@ -2,6 +2,9 @@
 
 UNAME=villager
 APP_PATH=/opt/hc-client
+TMATEURL=https://github.com/tmate-io/tmate/releases/download/2.4.0/
+TMATE64=tmate-2.4.0-static-linux-arm64v8.tar.xz
+TMATE32=tmate-2.4.0-static-linux-arm32v7.tar.xz
```

Comment:
Can we do a check to see if we are on a x64 or i386? Just in case someone uses an old laptop instead of a RPi?